lab2.js

```diff
@@ -117,9 +187,25 @@ assert((new Human()).sayHello(new Klingon()) === 'nuqneH',
 // assertions for each one (the assertions are how you
 // will test your code)
 //*********************************************************
-
+var cats = ['nacho', 'bob', 'alice', 'greta'];
 function lastLetterSort(stringArray) {
   function byLastLetter(a, b) {
+    if ((a.slice(-1)) > (b.slice(-1))) {
+      return 1;
+    }
+    if ((a.slice(-1)) < (b.slice(-1))) {
+      return -1;
+    }
+    return 0;
```

Comment:
this works. and if you want, you can just return a.slice(-1) - b.slice(-1). That will give you a positive number, negative number, or 0.